assets/js/123validations.js

```diff
@@ -0,0 +1,65 @@
+function content_for_step_is_valid(clickedButton){
```

Comment:
Identifier 'content_for_step_is_valid' is not in camel case.<br>'content_for_step_is_valid' is defined but never used.